### ISSUE TITLE
Error with minimum similarity 0

### ISSUE THREAD

**Issue by ScrappyCocco** [ScrappyCocco/HowLongToBeat-PythonAPI#4]
This seems to have caused a side effect where if you search for a game that is not found, you get a `HowLongToBeatEntry` with all `None` values. But only if you specify `input_minimum_similarity=0`.

```
from howlongtobeatpy import HowLongToBeat, HowLongToBeatEntry
results = HowLongToBeat().search('Thief: Gold')
print(len(results))
> 0
results = HowLongToBeat(input_minimum_similarity=0.4).search('Thief: Gold')
print(len(results))
> 0
results = HowLongToBeat(input_minimum_similarity=0).search('Thief: Gold')
print(len(results))
> 1
print(vars(results[0]))
> {'game_id': -1,
 'game_name': None,
 'game_image_url': None,
 'game_web_link': None,
 'gameplay_main': -1,
 'gameplay_main_unit': None,
 'gameplay_main_label': None,
 'gameplay_main_extra': -1,
 'gameplay_main_extra_unit': None,
 'gameplay_main_extra_label': None,
 'gameplay_completionist': -1,
 'gameplay_completionist_unit': None,
 'gameplay_completionist_label': None,
 'similarity': 0}
```

_Originally posted by @kparal in https://github.com/ScrappyCocco/HowLongToBeat-PythonAPI/issues/3#issuecomment-576176425_

**Comment by ScrappyCocco** [comment]
@kparal fixed in 0.1.12, commit [https://github.com/ScrappyCocco/HowLongToBeat-PythonAPI/commit/49611da3f6ab2f8035eba660b6319d4b159e98dc](https://github.com/ScrappyCocco/HowLongToBeat-PythonAPI/commit/49611da3f6ab2f8035eba660b6319d4b159e98dc)

Let me know if it works and if you need anything else
Best regards

**Comment by kparal** [comment]
Seems to be working fine. Thanks.
